statistics-reports.md

```diff
@@ -0,0 +1,193 @@
+                        "values": {
+                            "views": 0
+                        }
+                    },
+                    {
+                        "label": "May 2020",
+                        "type": "date",
+                        "id": "2020-05",
+                        "values": {
+                            "views": 3
+                        }
+                    }
+                ]
+            },
+            {
+                "id": "1911e8a4-6939-490c-b58b-a5d70f8d91fb_TotalDownloads",
+                "type": "usage-report",
+                "report-type": "TotalVisits",
+                "points": [
+                    {
+                        "label": "8d33bdfb-e7ba-43e6-a93a-f445b7e8a1e2",
+                        "type": "bitstream",
+                        "id": "8d33bdfb-e7ba-43e6-a93a-f445b7e8a1e2",
+                        "values": {
+                            "downloads": 8
+                        }
+                    }
+                ]
+            },
+            {
+                "id": "1911e8a4-6939-490c-b58b-a5d70f8d91fb_TopCountries",
+                "type": "usage-report",
+                "report-type": "TotalVisits",
+                "points": [
+                    {
+                        "label": "United States",
+                        "type": "country",
+                        "id": "US",
+                        "values": {
+                            "views": 2
+                        }
+                    },
+                    {
+                        "label": "China",
+                        "type": "country",
+                        "id": "CN",
+                        "values": {
+                            "views": 1
+                        }
+                    }
+                ]
+            }
+        ]
+    }
+}
+```
+
+Possible response status:
+* 200 OK - The DSpaceObject was found, and the data has been properly returned.
+* 400 Bad Request - The scope or scopeType parameter format is incorrect
+
+## Single statistic
+**GET /api/statistics/usage-reports/<:id>**
+
+This endpoint provides a specific statistic
+
+An example JSON response document to `/api/statistics/usage-reports/1911e8a4-6939-490c-b58b-a5d70f8d91fb_TopCountries`:
+```json
+{
+    "id": "1911e8a4-6939-490c-b58b-a5d70f8d91fb_TopCountries",
+    "type": "usage-report",
+    "report-type": "TotalVisits",
+    "points": [
+        {
+            "label": "United States",
+            "type": "country",
+            "id": "US",
+            "values": {
+                "views": 2
+            }
+        },
+        {
+            "label": "China",
+            "type": "country",
+            "id": "CN",
+            "values": {
+                "views": 1
+            }
+        }
+    ]
+}
+```
+
+Possible response status
+
+- 200 OK - The specific statistics data was found, and the data has been properly returned.
+- 404 Not Found - The specified ID was not found
```

Comment:
I suspect we also need an error code for access restricted/denied. Usage reports are not always public, see this set of configs: https://github.com/DSpace/DSpace/blob/master/dspace/config/modules/usage-statistics.cfg#L26-L31

Reply:
I've added this response status for the endpoints